src/test/java/org/javarosa/core/model/DynamicSelectUpdateTest.java

```diff
@@ -131,8 +131,8 @@ private static XFormsElement getSelectFromRepeatForm(String predicate) {
                             t("select"))))),
             body(
                 repeat("/data/repeat",
-                    input("value"),
-                    input("label")),
+                    input("/data/repeat/value"),
+                    input("/data/repeat/label")),
                 input("filter"),
```

Comment:
Looks like this one was missed (though another for the same name was caught below). I suspect there may be more. I'll try to catalogue them all so they can be updated together.

It's also unclear to me whether itemsets should be addressed the same way. E.g. should `"../repeat"` in the `select1Dynamic` call on the next line be `/data/repeat`? That might make sense here, but I'm less sure about extending that reasoning to relative itemset `nodeset`s **from within** repeats.